### PR TITLE
Add indonesian translation

### PR DESCRIPTION
**Pull request by ghost**
Indonesian translation

### CODE REVIEW

**Comment by haacked** [comment]
Thanks @fesuydev! Unfortunately, I am not capable of reviewing the translation. Do you know others who can provide a review?

**Comment by cwibowo** [comment]
Hi, I am a native Indonesian and I might be able to help. (Also I've been living abroad for two years so my English is OK).

I've read some of the translations and they look like literal translations.
For example:
`MAJOR version when you make incompatible API changes`
The current translation is:
`versi MAJOR ketika anda membuat perubahan kompatibilitas API`
which means:
`MAJOR version when you make changes to the API compatibility`

In my opinion it should be:
`versi MAJOR ketika anda membuat perubahan API yang tidak lagi cocok dengan versi sebelumnya`
which can be translated back to English to:
`MAJOR version when you make changes to the API that is no longer compatible with the previous version`

There are some English words that are not exist in Bahasa Indonesia, and vice versa.

Another example:
`MINOR version when you add functionality in a backwards-compatible manner`
Current translation:
`versi MINOR ketika anda menambah fungsionalitas pada gaya kompatibilitas sebelumnya`
which means:
`MINOR versions when you add functionality to the style of the previous compatibility`

In my opinion it should be:
`versi MINOR ketika anda menambahkan fungsionalitas yang masih sesuai dengan versi sebelumnya`
which means:
`MINOR version when you add functionality that is still compatible with previous versions`

Let me know if my translation makes sense, then I will continue with the other.
(Or maybe learn Bahasa Indonesia? It's a very easy language to learn 😄 )

**Review by adityapurwa — CHANGES_REQUESTED** [review]
Most of the sentence feels like generated from a translator and might be hard for native speaker to understand it without changing the sentence to fit Indonesian conversation style.

And the RFC 2119 translation to Indonesia is kind of miss-leading, such as MUST NOT translated to TIDAK HARUS is miss-leading, because TIDAK HARUS could be translated as MAY NOT.

File: lang/id/index.md [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+language: id
+---
+
+Semantic Versioning 2.0.0
+=============================
+
+Ringkasan
+-------
+
+Diberi nomor versi MAJOR.MINOR.PATCH, kenaikan pada:
```
Comment (adityapurwa) [inline lang/id/index.md:12]: Kenaikan pada is not a clear Indonesian word.

File: lang/id/index.md [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+label tambahan untuk pre-release dan build metadata tersedia sebagai ekstensi
+untuk format MAJOR.MINOR.PATCH.
+
+Pengantar
+------------
+
+Dalam dunia manajemen software terdapat tempat menakutkan yang disebut
+"dependency hell" Semakin besar sistem anda tumbuh dan semakin paket anda
+diintegrasikan ke dalam perangkat lunak anda, semakin besar kemungkinan anda
+untuk menemukan diri anda suatu hari pada lingkaran keputusasaan ini.
```
Comment (adityapurwa) [inline lang/id/index.md:27]: Direct translation makes this hard to grasp by Indonesian, try cutting some unnecessary word.

File: lang/id/index.md [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+
+Pada sistem dengan banyak dependensi, merilis versi paket baru dapat dengan cepat
+menjadi mimpi buruk. Jika spesifikasi dependensi terlalu ketat, anda akan masuk ke
+bahaya dari "version lock" (ketidakmampuan untuk meng-upgrade paket tanpa harus
+merilis versi baru dari setiap paket terkait). Jika dependensi
+ditentukan terlalu longgar, anda pasti akan digigit oleh "version promiscuity"
+(dengan asumsi kompatibilitas dengan versi yang lebih di masa mendatang lebih dari wajar).
+Dependency hell tempat anda berada ketika "version lock" dan/atau "version promiscuity"
+mencegah anda dari kemudahan dan keamanan pergerakan projek anda kedepan.
```
Comment (adityapurwa) [inline lang/id/index.md:36]: Might need to reorder the word to fit Indonesian conversation style, and this is a conclusion but does not include any conclusion word.

Probably can be changed as:

> Dua situasi di atas adalah "dependency hell" yang kita sebutkan, yang membuat kita sulit untuk mengembangkan project lebih lanjut.

File: lang/id/index.md [hunk 4]
```diff
@@ -0,0 +1,340 @@
+---
+title: Semantic Versioning 2.0.0
```
Comment (adityapurwa) [inline lang/id/index.md:2]: Probably use Indonesian translation for title "Versi Semantik"

File: lang/id/index.md [hunk 5]
```diff
@@ -0,0 +1,340 @@
...
+Kata kunci "HARUS", "TIDAK HARUS", "DIPERLUKAN", "AKAN", "TIDAK AKAN", "SEHARUSNYA",
+"TIDAK SEHARUSNYA", "DIREKOMENDASIKAN", "MUNGKIN", dan "OPSIONAL" pada dokumen ini harus
+ditafsirkan seperti yang dijelaskan dalam [RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+1. Perangkan lunak menggunakan Semantic Versioning HARUS menyatakan API publik. API ini
+bisa dinyatakan dalam kode itu sendiri atau ada ketepatan dalam dokumentasi.
+Bagaimanapun hal itu dikakukan, SEHARUSNYA tepat dan komprehensif.
+
+1. Penomoran versi normal HARUS mengambil bentuk X.Y.Z dimana X, Y, dan Z adalah
+bukan bilangan bulat negatif, dan TIDAK HARUS berisi nol terkemuka. X adalah
```
Comment (adityapurwa) [inline lang/id/index.md:68]: Should not translate English directly, word like "bukan bilangan bulat negatif" can be changed to "HARUS bilangan bulat positif"

File: lang/id/index.md [hunk 6]
```diff
@@ -0,0 +1,340 @@
...
+
+Kata kunci "HARUS", "TIDAK HARUS", "DIPERLUKAN", "AKAN", "TIDAK AKAN", "SEHARUSNYA",
+"TIDAK SEHARUSNYA", "DIREKOMENDASIKAN", "MUNGKIN", dan "OPSIONAL" pada dokumen ini harus
+ditafsirkan seperti yang dijelaskan dalam [RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+1. Perangkan lunak menggunakan Semantic Versioning HARUS menyatakan API publik. API ini
+bisa dinyatakan dalam kode itu sendiri atau ada ketepatan dalam dokumentasi.
+Bagaimanapun hal itu dikakukan, SEHARUSNYA tepat dan komprehensif.
+
+1. Penomoran versi normal HARUS mengambil bentuk X.Y.Z dimana X, Y, dan Z adalah
```
Comment (adityapurwa) [inline lang/id/index.md:67]: "HARUS mengambil" can be changed into "HARUS dalam bentuk"

File: lang/id/index.md [hunk 7]
```diff
@@ -0,0 +1,340 @@
...
+
+Saya menyebut sistem ini "Semantic Versioning." berdasarkan skema ini, penomoran versi
+dan cara mereka mengubah harus menyampaikan makna tentang kode yang mendasari dan apa yang
+telah dimodifikasi dari satu versi ke yang berikutnya.
+
+
+Spesifikasi Semantic Versioning (SemVer)
+------------------------------------------
+
+Kata kunci "HARUS", "TIDAK HARUS", "DIPERLUKAN", "AKAN", "TIDAK AKAN", "SEHARUSNYA",
```
Comment (adityapurwa) [inline lang/id/index.md:59]: "TIDAK HARUS" should be changed to TIDAK BOLEH so its not miss-leading, "TIDAK AKAN" may be changed to "TIDAK BISA"

File: lang/id/index.md [hunk 8]
```diff
@@ -0,0 +1,340 @@
...
+"TIDAK SEHARUSNYA", "DIREKOMENDASIKAN", "MUNGKIN", dan "OPSIONAL" pada dokumen ini harus
```
Comment (adityapurwa) [inline lang/id/index.md:60]: MUNGKIN can be changed into BISA

File: lang/id/index.md [hunk 9]
```diff
@@ -0,0 +1,340 @@
...
+ditentukan berdasarkan perbandingan setiap pengidentifikasi titik pemisah dari kiri ke kanan
+sampai perbedaan yang ditemukan sebagai berikut: pengidentifikasi yang terdiri dari angka saja
+dibandingkan dengan angka dan pengidentifikasi dengan huruf atau tanda hubung dibandingkan
+leksikal pada urutan ASCII. Pengidentifikasi numerik selalu memiliki prioritas yang lebih rendah
+dari pengidentifikasi non-numerik. Satu set besar bidang pre-release memiliki prioritas
+lebih tinggi dibanding satu set kecil, jika semua pengidentifikasi sebelumnya adalah sama.
+Contoh: 1.0.0-alpha < 1.0.0-alpha.1 < 1.0.0-alpha.beta < 1.0.0-beta <
+1.0.0-beta.2 < 1.0.0-beta.11 < 1.0.0-rc.1 < 1.0.0.
+
+Backus-Naur Form Grammar untuk Versi SemVer yang valid
```
Comment (adityapurwa) [inline lang/id/index.md:134]: I don't think this is neccessary

File: lang/id/index.md [hunk 10]
```diff
@@ -0,0 +1,340 @@
...
+               | "U" | "V" | "W" | "X" | "Y" | "Z" | "a" | "b" | "c" | "d"
+               | "e" | "f" | "g" | "h" | "i" | "j" | "k" | "l" | "m" | "n"
+               | "o" | "p" | "q" | "r" | "s" | "t" | "u" | "v" | "w" | "x"
+               | "y" | "z"
+
+
+Mengapa menggunakan Semantic Versioning?
+----------------------------
+
+Ini bukan ide baru atau revolusioner. Bahkan, anda mungkin sudah melakukan sesuatu
```
Comment (adityapurwa) [inline lang/id/index.md:203]: Probably can be changed to:

> Bisa jadi anda sudah pernah menerapkan hal seperti ini sebelumnya.

File: lang/id/index.md [hunk 11]
```diff
@@ -0,0 +1,340 @@
...
+Ladder memiliki versi 3.1.0. Sejak Firetruck menggunakan beberapa fungsionalitas
+yang diperkenalkan pada 3.1.0, anda bisa dengan aman menentukan dependensi
+Ladder sebagai lebih dari atau sama dengan 3.1.0 namun kurang dari 4.0.0. Sekarang, ketika
+Ladder versi 3.1.1 dan 3.2.0 menjadi tersedia, anda dapat merilis mereka pada
+sistem manajemem paket dan tahu mereka itu akan kompatibel dengan ketergantungan
+software yang ada.
+
+Sebagai pengembang bertanggung jawab anda tentu saja ingin memverifikasi bahwa setiap
+Paket upgrade berfungsi seperti yang diiklankan. Dunia nyata adalah tempat yang berantakan;
+tidak ada yang bisa kita lakukan tentang itu tapi waspada.
```
Comment (adityapurwa) [inline lang/id/index.md:223]: > Meski kenyataanya kadang tidak seperti itu, paling tidak kita bisa berhati-hati dalam mengatur versi.

File: lang/id/index.md [hunk 12]
```diff
@@ -0,0 +1,340 @@
...
+---
+
+### Bagaimana saya harus berurusan dengan revisi dalam tahap pengembangan awal 0.y.z?
+
+Hal paling sederhana untuk dilakukan adalah mulai rilis pengembangan awal Anda di 0.1.0
+dan kemudian kenaikan versi minor untuk setiap rilis berikutnya.
+
+### Bagaimana saya tahu kapan harus merilis 1.0.0?
+
+Jika perangkat lunak Anda sedang digunakan dalam produksi, mestinya mungkin sudah menjadi
```
Comment (adityapurwa) [inline lang/id/index.md:245]: > Jika perangkat lunak Anda sudah digunakan dalam produksi, seharusnya sudah bisa disebut versi 1.0.0

File: lang/id/index.md [hunk 13]
```diff
@@ -0,0 +1,340 @@
...
+1.0.0. Jika Anda memiliki API yang stabil di mana pengguna telah tergantung, anda seharusnya
+menjadi 1.0.0. Jika Anda khawatir banyak tentang kompatibilitas sebelumnya, anda harus
+mestinya sudah menjadi 1.0.0.
+
+### Apakah ini tidak menyurutkan perkembangan pesat dan iterasi cepat?
```
Comment (adityapurwa) [inline lang/id/index.md:250]: Bukankah ini akan menghambat pengembangan?

File: lang/id/index.md [hunk 14]
```diff
@@ -0,0 +1,340 @@
...
+
+Versi Major nol adalah semua tentang perkembangan pesat. Jika Anda mengubah API
+setiap hari anda harus tetap dalam versi 0.y.z atau memisahkan
+cabang development yang bekerja pada versi utama berikutnya.
+
+### Jika segala perubahan kecil sebelumnya tidak kompatibel pada API publik memerlukan benturan versi major, akankah saya berakhir pada versi 42.0.0 sangat cepat?
+
+Ini adalah pertanyaan dari development yang bertanggung jawab dan pandangan ke depan. Perubahan
+tidak kompatibel tidak harus diperkenalkan enteng untuk software yang memiliki banyak
```
Comment (adityapurwa) [inline lang/id/index.md:259]: > Perubahan harusnya diatur dengan hati-hati supaya tidak terlalu banyak versi major karena terburu-terburu ingin merilis versi.

File: lang/id/index.md [hunk 15]
```diff
@@ -0,0 +1,340 @@
...
+seringkali diperlukan untuk membuat kemajuan ke depan. Ketika Anda mengusangkan bagian dari
+API publik anda, anda harus melakukan dua hal: (1) informasi dokumentasi anda untuk membiarkan
+pengguna tahu tentang perubahan, (2) mengeluarkan rilis minor baru dengan keusangan tersebut
+di tempat. Sebelum anda benar-benar menghapus fungsi dalam rilis utama baru
+harus ada setidaknya satu rilis minor yang berisi keusangan sehingga
+pengguna dapat dengan lancar transisi ke API baru.
+
+### Apakah SemVer memiliki batas ukuran pada string versi?
+
+Tidak, tapi gunakan penilaian yang baik. 255 karakter string untuk versi mungkin berlebihan,
```
Comment (adityapurwa) [inline lang/id/index.md:313]: > Tidak, tapi gunakan secukupnya. Versi dengan 255 karakter terlalu berlebihan, dan menyulitkan orang untuk membacanya.

**Review by adityapurwa — CHANGES_REQUESTED** [review]
I might miss some parts, please recheck the whole content and make sure to use Indonesian context for translation.

**Comment by adityapurwa** [comment]
@fesuydev Closed for fix to create another pull request?

**Comment by ghost** [comment]
@adityapurwa sorry, I close this pull request cause I want delete this account

**Pull request closed without merging**